src/components/storage/InstallationDestination.jsx

```diff
@@ -433,7 +433,7 @@ export const InstallationDestination = ({
                                 : _("No usable disks detected")
                         )}
                     {rescanDisksButton}
-                    {!isBootIso && <ModifyStorage idPrefix={idPrefix} onCritFail={onCritFail} onRescan={onClickRescan} />}
```

Comment:
I had to remove the `isBootIso` check only so that I could manually test the "Modify storage" button, since it the flag was true in my test setup. I don't understand at all what it is about, so removing this check as part of the Cockpit integration is probably wrong.